python/services/dataservice/dmod/dataservice/service_settings.py

```diff
@@ -1,20 +1,203 @@
+    """Set the appropriate listening host name or address value (NOTE: must match SSL cert)"""
+    port: Port = Field(3012)
+    """Set the appropriate listening port value"""
+
+    cert_path: Optional[FilePath] = None
+    """Specify path for a particular SSL certificate file to use"""
+    key_path: Optional[FilePath] = None
+    """Specify path for a particular SSL private key file to use"""
+    ssl_dir: Optional[DirectoryPath] = None
+    """Change the base directory when using SSL certificate and key files with default names"""
+
+    object_store_host: str = "minio-proxy"
+    """Set hostname for connection to object store"""
+    object_store_port: Port = Port(9000)
+    """Set port for connection to object store"""
+
+    # TODO: consider making this a pydantic.SecretStr
+    object_store_exec_user_name: str
+    """Object store user access key"""
+
+    # TODO: consider making this a pydantic.SecretStr
+    object_store_exec_user_passwd: str
+    """Object store user secret key"""
+
+    redis_host: str = "myredis"
+    """Set the host value for making Redis connections"""
+    redis_port: Port = Port(6379)
+    """Set the port value for making Redis connections"""
+
+    # TODO: consider making this a pydantic.SecretStr
+    redis_pass: str = "noaaOwp"
+    """Set the password value for making Redis connections"""
+
+    pycharm_debug: bool = False
+    """Activate Pycharm remote debugging support"""
+
+    @classmethod
+    def usage(cls, verbose: bool = False) -> str:
+        """
+        Return a formatted string with service configuration usage information.
+        Increase the amount of detail included using the verbose flag.
+        """
+
+        header = f"""
+Usage: Provide listed service configuration variables as either environment
+variables, in a '.env' file, as a 'secret' in {cls.Config.secrets_dir!r}, or as
+a mixture of the aforementioned options.
+
+Configuration option names are case insensitive.
+
+Configuration Variable:
+"""
+        verbose_header = f"""
+Usage: Provide listed service configuration variables as either environment
+variables, in a '.env' file, as a 'secret' in {cls.Config.secrets_dir!r}, or as
+a mixture of the aforementioned options.
+
+Configuration option names are case insensitive.
+
+boolean values are case insensitive.
+valid values are:
+    0, off, f, false, n, no, 1, on, t, true, y, yes
+
+A '.env' file must be utf-8 encoded and follow syntax rules:
+- lines beginning with `#` are treated as comments
+- blank lines are ignored
+- each line represents a key-value pair. values can optionally be quoted.
+  for example:
+      PORT=8080
+      PORT='8080'
+      PORT="8080"
+For more in-depth information, see these resources:
+- https://docs.docker.com/compose/environment-variables/env-file/#syntax
+- https://github.com/theskumar/python-dotenv?tab=readme-ov-file#file-format
+
+A secret's filename must be the name of a configuration variable.
```

Comment:
Similar to the above:

```suggestion
A secret's filename must match the name of a configuration variable (case insensitive).
```